### PR TITLE
Add course instructions

### PR DESCRIPTION
**Pull request by ankitjavalkar**
image

### CODE REVIEW

File: yaksh/models.py [hunk 1]
```diff
@@ -146,6 +146,7 @@ class Course(models.Model):
     created_on = models.DateTimeField(auto_now_add=True)
     teachers = models.ManyToManyField(User, related_name='teachers')
     is_trial = models.BooleanField(default=False)
+    instructions = models.TextField(null=True, blank=True)
```
Comment (maheshgudi) [inline yaksh/models.py]: please add a default value or none. This makes it easy for migrations.

File: yaksh/templates/yaksh/quizzes_user.html [hunk 2]
```diff
@@ -60,6 +60,21 @@
             </table>
         {% endif %}
         </div>
+
+        <div class="row">
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/quizzes_user.html]: add a if instructions then display in the template condition.

File: yaksh/templates/yaksh/quizzes_user.html [hunk 3]
```diff
@@ -60,6 +60,21 @@
...
+        <div class="panel-group">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h4 class="panel-title">
+                <a data-toggle="collapse" href="#collapse{{ forloop.counter }}">Course Instructions</a>
+              </h4>
+            </div>
+            <div id="collapse{{ forloop.counter }}" class="panel-collapse collapse">
+              <div class="panel-body"> {{ course.instructions }}</div>
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/quizzes_user.html]: could the course instructions be rendered as html.

Comment (ankitjavalkar) [inline yaksh/templates/yaksh/quizzes_user.html]: We can add HTML tags to render as HTML

File: yaksh/templates/yaksh/quizzes_user.html (hunk 2 above)
Comment (ankitjavalkar) [inline yaksh/templates/yaksh/quizzes_user.html]: Done

File: yaksh/models.py (hunk 1 above)
Comment (ankitjavalkar) [inline yaksh/models.py]: Done, default is none and is only rendered if Instructions is a non Null value

**Comment by ankitjavalkar** [comment]
Issues addressed, mergeable.

**Pull request merged**